R/get_df_format.R

```diff
@@ -0,0 +1,60 @@
+#' @title Determine format of data frame
+#'
+#' @description
+#' Helper function to determine whether a supplied data frame is of any of the
+#' \code{\link{orgsurveyr-data-formats}} defined in the package.
+#'
+#' @param df query data frame
+#' @return string of data frame type
+#' @export get_df_format
+#'
+#' @examples
+#' data(tg_org_indiv_df)
+#' get_df_format(tg_org_indiv_df)
+#'
+#' data(tg_org_indiv_minimal_df)
+#' get_df_format(tg_org_indiv_minimal_df)
+#'
+#' data(tg_org_indiv_tall_df)
+#' get_df_format(tg_org_indiv_tall_df)
+#'
+#' data(tg_org_summarised_df)
+#' get_df_format(tg_org_summarised_df)
+#'
+#' data(mtcars)
+#' get_df_format(mtcars)
+get_df_format <- function(df) {
+
+  compare_df <- function(query_df, template_df, format_id, subset_ok = FALSE) {
+    data(list=template_df)
```

Comment:
Consider adding
  `importFrom("utils", "data")`
to your NAMESPACE file.

Reply:
Review if `data()` is appropriate by referring to the help section. Specifically see section ‘Good practice’ in ‘?data’.